### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by nikitashrestha**
- fixed link issues
- updated pdf name for download

### CODE REVIEW

File: src/components/pdf/pdf.styles.js [hunk 1]
```diff
@@ -5,6 +5,18 @@ Font.register({
...
 });
 
+// Register hyphenation callback.
+// In this example, we enable words to break in half.
+Font.registerHyphenationCallback(word => {
+  const middle = Math.floor(word.length / 2);
+  const parts = word.length === 1 ? [word] : [word.substr(0, middle), word.substr(middle)];
+
+  // Check console to see words parts.
+  console.log(word, parts);
```
Comment (sagunji) [inline src/components/pdf/pdf.styles.js:15]: remove console.log

File: src/utilities/string/checkProtocol.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { protocolRegEx, http } from '~/common/constants';
+
+/**
```
Comment (sagunji) [inline src/utilities/string/checkProtocol.js:3]: or http too, right?

Comment (nikitashrestha) [reply src/utilities/string/checkProtocol.js:3]: i am unclear on your comment.can you elaborate?

File: src/components/pdf/pdf.styles.js (hunk 1 above)
Comment (nikitashrestha) [reply src/components/pdf/pdf.styles.js:15]: done

**Pull request merged**